website/config/custom.js

```diff
@@ -225,14 +233,22 @@ module.exports.custom = {
     'website/config/custom.js': ['eashaw', 'mikermcneil'],// (« for changing whose changes automatically approve and unfreeze relevant PRs changing given paths)
 
     // Handbook
-    'handbook': ['mike-j-thomas', 'eashaw', 'mikermcneil', 'zwass'],// (default for handbook)
-    'handbook/company': 'mikermcneil',
-    'handbook/company/ceo.md': ['sampfluger88', 'mikermcneil'],
-    'handbook/business-operations': ['jostableford', 'hollidayn', 'mikermcneil'],
-    'handbook/engineering': ['zwass', 'lukeheath'],
-    'handbook/product': ['noahtalerman', 'zhumo'],
-    'handbook/customers': ['alexmitchelliii','zayhanlon','dherder'],
-    'handbook/marketing': ['jarodreyes', 'mike-j-thomas', 'spokanemac'],
+     'handbook/': 'mikermcneil',
+    // Everyone can contribute to the handbook, and page DRIs are expected to keep their pages up to date.  
```

Comment:
"See [https://github.com/fleetdm/fleet/pull/13195/files](https://github.com/fleetdm/fleet/pull/13195/files)"
Instead of dup comment.